### ISSUE TITLE
GET requests fail due to uri encoding

### ISSUE THREAD

**Issue by michaelCaleyWhaley** [apollo-server-integrations/apollo-server-integration-aws-lambda#120]
Recently setup a lambda behind an ALB and attempted to make GET requests resulting in error message 'The variables search parameter contains invalid JSON.'

_Example query params_
```
queryStringParameters: {
          operationName: 'product',
          query: example query',
          variables:
'%7B%22productId%22%3A%22756817%22%2C%22countryCode%22%3A%22GB%22%2C%22currencyCode%22%3A%22GBP%22%7D',
        }
```

decoding the value via decodeURIComponent solved the issue. 

```
parseQueryParams(event) {
      const params = new URLSearchParams();
      for (const [key, value] of Object.entries((event.queryStringParameters as Record<string, string>) ?? {})) {
        params.append(key, decodeURIComponent(value) ?? '');
      }
      for (const [key, value] of Object.entries(
        (event.multiValueQueryStringParameters as Record<string, string> | undefined) ?? {},
      )) {
        for (const v of value ?? []) {
          params.append(key, v);
        }
      }
      return params.toString();
    }
```

**Comment by BlenderDude** [comment]
Thanks for reporting this, seems to be an oversight in testing. I'll get a fix implemented and pushed here in the next day or so!

**Comment by BlenderDude** [comment]
Fixed in #122
